.github/workflows/Tests.yaml

```diff
@@ -32,7 +32,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Checkout repository
-        uses: actions/checkout@v1
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
I think this is the only place in this PR where you need the `fetch-depth: 0`. Please remove the others.

Reply:
I'd rather get this merged so we can restart tests... This is also somewhat annoying to debug. I'm also not convinced we don't need it for clang-tidy...

Reply:
Sure, I'm fine with merging this now and opening another PR that removes the `fetch-depth`s.